.github/workflows/test.yaml

```diff
@@ -174,6 +182,54 @@ jobs:
         if: failure()
         run: bash .github/slack/notify-failure.sh core-features ${{ matrix.feature }}
 
+  migrate:
+    needs: start-time
+
+    strategy:
+      fail-fast: false
+      matrix:
+        engine: ['napi', 'binary']
+        feature: [
+          db-seed
+        ]
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    concurrency: ${{ github.run_id }}-migrate
```

Comment:
This will cancel all runs except 2 unfortunately. We should probably modify the database URL in the test, so it can create a new DB for each test matrix combination, then no `concurrency` necessary.